lisp/ess-r-mode.el

```diff
@@ -711,84 +711,53 @@ Executed in process buffer."
         (cons val long-path)
       val)))
 
+(defmacro ess-r-create-runner (arg)
+  "Create a function ARG.
+This function starts R with `inferior-R-version' set to ARG."
+  `(let ((version ,arg))
```

Comment:
Let's rename ARG into RUNNER-FN or RUNNER, or  FN.  `version` is not suggestive either. How about dropping the let binding and doing `(intern ,arg)` directly?